Dockerfile

```
@@ -0,0 +1,9 @@
+FROM xshuden/alppython3
```

```suggestion
FROM python:3
```

need to check whether `git` is installed

---

In alppython3 you can check it on the git loaded hub.

---

thanks for the info. i'll leave it as is then.